src/main/java/com/timelogr/enterprise/dto/client.java

```diff
@@ -1,4 +1,4 @@
-package com.timelogr.enterprise;
+package com.timelogr.enterprise.dto;
 
 public class client {
```

Comment:
As long as you are in here, make this UpperCamelCase as well.  Should be Client, not client.